### PR TITLE
New MMDP icons

### PR DESCRIPTION
**Pull request by Browun**
#### Checklist

- [ ] Includes tests
- [ ] Update documentation

#### Changes proposed in this pull request:

Addind the following icons:
- Table: virtual
- Table: virtual add
- Table: add
- Table: list add

#### Reviewers should focus on:

Ensuring everything renders ok and the right files were changed.

#### Screenshot

Screenshot 2024-11-14 at 4 11 43 PM

### CODE REVIEW

File: packages/icons/icons.json [hunk 1]
```diff
@@ -4541,5 +4541,33 @@
         "tags": "arrow, box, object, counter-clockwise, ccw, left",
         "group": "interface",
         "codepoint": 62345
+    },
+    {
+        "displayName": "Table: add",
+        "iconName": "th-add",
```
Comment (ggdouglas) [inline packages/icons/icons.json:4547]: Why are the icon names and display names different here? ("table" vs. "th"). I see that there are some other "th" icons that look somewhat similar already. Is there something about these icons that is specific to table headers?

File: resources/icons/16px/th-add.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M13 0C12.4477 0 12 0.447715 12 1V2H11C10.4477 2 10 2.44772 10 3C10 3.55228 10.4477 4 11 4H12V5C12 5.55228 12.4477 6 13 6C13.5523 6 14 5.55228 14 5V4H15C15.5523 4 16 3.55228 16 3C16 2.44772 15.5523 2 15 2H14V1C14 0.447715 13.5523 0 13 0Z" fill="#5F6B7C"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M8.76391 5C8.28886 4.46924 8 3.76835 8 2.99998C8 2.23163 8.28885 1.53075 8.76388 1H1C0.4 1 0 1.5 0 2V14C0 14.6 0.4 15 1 15H15C15.6 15 16 14.6 16 14V5.82927C15.9576 5.84426 15.9148 5.85832 15.8715 5.87143C15.4982 7.10329 14.3538 8 13 8C12.1115 8 11.3132 7.61375 10.7639 7H7V5H8.76391ZM6 5H2V7H6V5ZM6 8H2V10H6V8ZM7 10H14V8H13H7V10ZM6 11H2V13H6V11ZM7 13H14V11H7V13Z" fill="#5F6B7C"/>
+</svg>
```
Comment (ggdouglas) [inline resources/icons/16px/th-add.svg:4]:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
  <path d="M13 0c-.5523 0-1 .447715-1 1v1h-1c-.5523 0-1 .44772-1 1s.4477 1 1 1h1v1c0 .55228.4477 1 1 1s1-.44772 1-1V4h1c.5523 0 1-.44772 1-1s-.4477-1-1-1h-1V1c0-.552285-.4477-1-1-1Z"/>
  <path d="M8.76391 5H7v2h3.7639c.5493.61375 1.3476 1 2.2361 1 1.3538 0 2.4982-.89671 2.8715-2.12857.0433-.01311.0861-.02717.1285-.04216V14c0 .6-.4 1-1 1H1c-.6 0-1-.4-1-1V2c0-.5.4-1 1-1h7.76388C8.28885 1.53075 8 2.23163 8 2.99998 8 3.76835 8.28886 4.46924 8.76391 5ZM6 5H2v2h4V5Zm0 3H2v2h4V8Zm1 2h7V8H7v2Zm-1 1H2v2h4v-2Zm1 2h7v-2H7v2Z"/>
</svg>
```

File: resources/icons/16px/th-list-add.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M12 1C12 0.447715 12.4477 0 13 0C13.5523 0 14 0.447715 14 1V2H15C15.5523 2 16 2.44772 16 3C16 3.55228 15.5523 4 15 4H14V5C14 5.55228 13.5523 6 13 6C12.4477 6 12 5.55228 12 5V4H11C10.4477 4 10 3.55228 10 3C10 2.44772 10.4477 2 11 2H12V1ZM8 2.99998C8 3.76835 8.28886 4.46924 8.76391 5H6H2V7H6H10.7639C11.3132 7.61375 12.1115 8 13 8H7H2V10H7H14V8H13C14.3538 8 15.4982 7.10329 15.8715 5.87143C15.9148 5.85832 15.9576 5.84426 16 5.82927V14C16 14.6 15.6 15 15 15H1C0.4 15 0 14.6 0 14V2C0 1.5 0.4 1 1 1H8.76388C8.28885 1.53075 8 2.23163 8 2.99998ZM7 13H2V11H7H14V13H7Z" fill="#5F6B7C"/>
+</svg>
```
Comment (ggdouglas) [inline resources/icons/16px/th-list-add.svg:3]:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
  <path d="M12 1c0-.552285.4477-1 1-1s1 .447715 1 1v1h1c.5523 0 1 .44772 1 1s-.4477 1-1 1h-1v1c0 .55228-.4477 1-1 1s-1-.44772-1-1V4h-1c-.5523 0-1-.44772-1-1s.4477-1 1-1h1V1ZM8 2.99998C8 3.76835 8.28886 4.46924 8.76391 5H2v2h8.7639c.5493.61375 1.3476 1 2.2361 1H2v2h12V8h-1c1.3538 0 2.4982-.89671 2.8715-2.12857.0433-.01311.0861-.02717.1285-.04216V14c0 .6-.4 1-1 1H1c-.6 0-1-.4-1-1V2c0-.5.4-1 1-1h7.76388C8.28885 1.53075 8 2.23163 8 2.99998ZM7 13h7v-2H2v2h5Z"/>
</svg>
```

File: resources/icons/16px/th-virtual-add.svg [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+</svg>
```
Comment (ggdouglas) [inline resources/icons/16px/th-virtual-add.svg:3]:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
  <path d="M12 1c0-.207109.063-.399512.1708-.559113C12.3505.174886 12.6548 0 13 0c.5523 0 1 .447715 1 1v1h1c.1838 0 .3561.04961.5041.13617.1299.07597.2411.18039.3251.30472.1078.1596.1708.352.1708.55911 0 .21962-.0708.42271-.1908.58768-.0875.12024-.2011.22024-.3326.29163C15.3349 3.95628 15.1726 4 15 4h-1v1c0 .20711-.063.39951-.1708.55911-.0825.12217-.1914.22512-.3184.30073C13.3613 5.94887 13.1866 6 13 6c-.1381 0-.2696-.02798-.3892-.07858-.1673-.07077-.3114-.18577-.4175-.33035C12.0718 5.4255 12 5.22113 12 5V4h-1c-.1725 0-.3348-.04368-.4765-.12059-.1417-.07698-.2628-.18724-.3527-.3203C10.063 3.39951 10 3.20711 10 3c0-.22715.0757-.43661.2033-.6045.1051-.13828.2454-.24836.4075-.31692C10.7304 2.02798 10.8619 2 11 2h1V1Zm2 6.82928c.8948-.31628 1.5947-1.04505 1.8716-1.95785.0432-.01311.0859-.02716.1284-.04216V14c0 .6-.3999 1-1 1H1c-.552246 0-1-.4477-1-1V2c0-.5.399902-1 1-1h7.76392C8.28882 1.53075 8 2.23163 8 2.99998 8 3.76834 8.28882 4.46924 8.76392 5H2v8h12V7.82928ZM7.3606 11.7205 5 9.36267V10c0 .5523-.44775 1-1 1s-1-.4477-1-1V7c0-.55229.44775-1 1-1h3c.55225 0 1 .44771 1 1s-.44775 1-1 1h-.64478l2.36499 2.3624c.3733.373.37305.9838-.00097 1.3572-.37378.3735-.98535.3739-1.35864.0009Z"/>
</svg>
```

File: resources/icons/16px/th-virtual.svg [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M7 8C7.55225 8 8 7.55229 8 7C8 6.44771 7.55225 6 7 6H4C3.44775 6 3 6.44771 3 7V10C3 10.5523 3.44775 11 4 11C4.55225 11 5 10.5523 5 10V9.36267L7.3606 11.7205C7.73389 12.0935 8.34546 12.0931 8.71924 11.7196C9.09326 11.3462 9.09351 10.7354 8.72021 10.3624L6.35522 8H7Z" fill="#5F6B7C"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M1 1C0.399902 1 0 1.5 0 2V14C0 14.5523 0.447754 15 1 15H15C15.6001 15 16 14.6 16 14V2C16 1.44772 15.5522 1 15 1H1ZM14 13H2V5H14V13Z" fill="#5F6B7C"/>
+</svg>
```
Comment (ggdouglas) [inline resources/icons/16px/th-virtual.svg:4]:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
  <path d="M7 8c.55225 0 1-.44771 1-1s-.44775-1-1-1H4c-.55225 0-1 .44771-1 1v3c0 .5523.44775 1 1 1s1-.4477 1-1v-.63733l2.3606 2.35783c.37329.373.98486.3726 1.35864-.0009.37402-.3734.37427-.9842.00097-1.3572L6.35522 8H7Z"/>
  <path d="M1 1c-.600098 0-1 .5-1 1v12c0 .5523.447754 1 1 1h14c.6001 0 1-.4 1-1V2c0-.55228-.4478-1-1-1H1Zm13 12H2V5h12v8Z"/>
</svg>
```

File: resources/icons/20px/th-add.svg [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M15 1C15 0.447708 15.4478 0 16 0C16.5522 0 17 0.447708 17 1V3H19C19.5522 3 20 3.44771 20 4C20 4.55229 19.5522 5 19 5H17V7C17 7.55229 16.5522 8 16 8C15.4478 8 15 7.55229 15 7V5H13C12.4478 5 12 4.55229 12 4C12 3.44771 12.4478 3 13 3H15V1Z" fill="#5F6B7C"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M19 7C19 8.24265 18.2444 9.30882 17.1677 9.76425C17.1382 9.77672 17.1084 9.78873 17.0784 9.80028C17.0525 9.8103 17.0264 9.81998 17 9.82928C16.6873 9.93985 16.3506 10 16 10C15.1116 10 14.3132 9.61375 13.7639 9H8V6H10.7639C10.2888 5.46924 10 4.76836 10 4C10 2.34314 11.3433 1 13 1H1C0.399902 1 0 1.5 0 2V18C0 18.5523 0.447754 19 1 19H18C18.5522 19 19 18.5523 19 18V7ZM16 10H17V13H8V10H16ZM7 9V6H2V9H7ZM2 10H7V13H2V10ZM2 14H7V17H2V14ZM17 14H8V17H17V14Z" fill="#5F6B7C"/>
+</svg>
```
Comment (ggdouglas) [inline resources/icons/20px/th-add.svg:4]:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
  <path d="M15 1c0-.552292.4478-1 1-1s1 .447708 1 1v2h2c.5522 0 1 .44771 1 1s-.4478 1-1 1h-2v2c0 .55229-.4478 1-1 1s-1-.44771-1-1V5h-2c-.5522 0-1-.44771-1-1s.4478-1 1-1h2V1Z"/>
  <path d="M19 7v11c0 .5523-.4478 1-1 1H1c-.552246 0-1-.4477-1-1V2c0-.5.399902-1 1-1h12c-1.6567 0-3 1.34314-3 3 0 .76836.2888 1.46924.7639 2H8v3h5.7639c.5493.61375 1.3477 1 2.2361 1 .3506 0 .6873-.06015 1-.17072.0264-.0093.0525-.01898.0784-.029.03-.01155.0598-.02356.0893-.03603C18.2444 9.30882 19 8.24265 19 7Zm-3 3H8v3h9v-3h-1ZM7 9V6H2v3h5Zm-5 1v3h5v-3H2Zm0 4v3h5v-3H2Zm15 0H8v3h9v-3Z"/>
</svg>
```

File: resources/icons/20px/th-list-add.svg [hunk 7]
```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M16 0C15.4478 0 15 0.447708 15 1V3H13C12.4478 3 12 3.44771 12 4C12 4.55229 12.4478 5 13 5H15V7C15 7.55229 15.4478 8 16 8C16.5522 8 17 7.55229 17 7V5H19C19.5522 5 20 4.55229 20 4C20 3.44771 19.5522 3 19 3H17V1C17 0.447708 16.5522 0 16 0Z" fill="#5F6B7C"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M17.1677 9.76425C18.2444 9.30882 19 8.24265 19 7V18C19 18.5523 18.5522 19 18 19H1C0.447754 19 0 18.5523 0 18V2C0 1.5 0.399902 1 1 1H13C11.3433 1 10 2.34314 10 4C10 4.76836 10.2888 5.46924 10.7639 6H2V9H13.7639C14.3132 9.61375 15.1116 10 16 10H2V13H17V10H16C16.3506 10 16.6873 9.93985 17 9.82928C17.0264 9.81998 17.0525 9.8103 17.0784 9.80028C17.1084 9.78873 17.1382 9.77672 17.1677 9.76425ZM2 14V17H17V14H2Z" fill="#5F6B7C"/>
+</svg>
```
Comment (ggdouglas) [inline resources/icons/20px/th-list-add.svg:4]:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
  <path d="M16 0c-.5522 0-1 .447708-1 1v2h-2c-.5522 0-1 .44771-1 1s.4478 1 1 1h2v2c0 .55229.4478 1 1 1s1-.44771 1-1V5h2c.5522 0 1-.44771 1-1s-.4478-1-1-1h-2V1c0-.552292-.4478-1-1-1Z"/>
  <path d="M17.1677 9.76425C18.2444 9.30882 19 8.24265 19 7v11c0 .5523-.4478 1-1 1H1c-.552246 0-1-.4477-1-1V2c0-.5.399902-1 1-1h12c-1.6567 0-3 1.34314-3 3 0 .76836.2888 1.46924.7639 2H2v3h11.7639c.5493.61375 1.3477 1 2.2361 1H2v3h15v-3h-1c.3506 0 .6873-.06015 1-.17072.0264-.0093.0525-.01898.0784-.029.03-.01155.0598-.02356.0893-.03603ZM2 14v3h15v-3H2Z"/>
</svg>
```

File: resources/icons/20px/th-virtual-add.svg [hunk 8]
```diff
@@ -0,0 +1,5 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M15 1C15 0.447715 15.4477 0 16 0C16.5523 0 17 0.447715 17 1V3H19C19.5523 3 20 3.44772 20 4C20 4.55228 19.5523 5 19 5H17V7C17 7.55228 16.5523 8 16 8C15.4477 8 15 7.55228 15 7V5H13C12.4477 5 12 4.55228 12 4C12 3.44772 12.4477 3 13 3H15V1Z" fill="#5F6B7C"/>
+<path d="M1 1H13C11.3433 1 10 2.34315 10 4C10 4.76836 10.2888 5.46925 10.7639 6H2V17H17V9.82928C18.165 9.41743 19 8.3062 19 7V18C19 18.5523 18.5522 19 18 19H1C0.447754 19 0 18.5523 0 18V2C0 1.5 0.399902 1 1 1Z" fill="#5F6B7C"/>
+<path d="M3 8C3 7.44771 3.44775 7 4 7H9C9.55225 7 10 7.44771 10 8C10 8.55229 9.55225 9 9 9H6.41431L11.707 14.2928C12.0977 14.6833 12.0977 15.3165 11.707 15.707C11.3164 16.0976 10.6833 16.0976 10.2927 15.707L5 10.4143V13C5 13.5523 4.55225 14 4 14C3.44775 14 3 13.5523 3 13V8Z" fill="#5F6B7C"/>
+</svg>
```
Comment (ggdouglas) [inline resources/icons/20px/th-virtual-add.svg:5]:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
  <path d="M15 1c0-.552285.4477-1 1-1s1 .447715 1 1v2h2c.5523 0 1 .44772 1 1s-.4477 1-1 1h-2v2c0 .55228-.4477 1-1 1s-1-.44772-1-1V5h-2c-.5523 0-1-.44772-1-1s.4477-1 1-1h2V1Z"/>
  <path d="M1 1h12c-1.6567 0-3 1.34315-3 3 0 .76836.2888 1.46925.7639 2H2v11h15V9.82928C18.165 9.41743 19 8.3062 19 7v11c0 .5523-.4478 1-1 1H1c-.552246 0-1-.4477-1-1V2c0-.5.399902-1 1-1Z"/>
  <path d="M3 8c0-.55229.44775-1 1-1h5c.55225 0 1 .44771 1 1s-.44775 1-1 1H6.41431l5.29269 5.2928c.3907.3905.3907 1.0237 0 1.4142-.3906.3906-1.0237.3906-1.4143 0L5 10.4143V13c0 .5523-.44775 1-1 1s-1-.4477-1-1V8Z"/>
</svg>
```

File: resources/icons/20px/th-virtual.svg [hunk 9]
```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M3 8C3 7.44771 3.44775 7 4 7H9C9.55225 7 10 7.44771 10 8C10 8.55229 9.55225 9 9 9H6.41431L11.707 14.2928C12.0977 14.6833 12.0977 15.3165 11.707 15.707C11.3164 16.0976 10.6833 16.0976 10.2927 15.707L5 10.4143V13C5 13.5523 4.55225 14 4 14C3.44775 14 3 13.5523 3 13V8Z" fill="#5F6B7C"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M20 18C20 18.5523 19.5522 19 19 19H1C0.447754 19 0 18.5523 0 18V2C0 1.5 0.399902 1 1 1H19C19.5522 1 20 1.44772 20 2V18ZM18 17H2V6H18V17Z" fill="#5F6B7C"/>
+</svg>
```
Comment (ggdouglas) [inline resources/icons/20px/th-virtual.svg:4]:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
  <path d="M3 8c0-.55229.44775-1 1-1h5c.55225 0 1 .44771 1 1s-.44775 1-1 1H6.41431l5.29269 5.2928c.3907.3905.3907 1.0237 0 1.4142-.3906.3906-1.0237.3906-1.4143 0L5 10.4143V13c0 .5523-.44775 1-1 1s-1-.4477-1-1V8Z"/>
  <path d="M20 18c0 .5523-.4478 1-1 1H1c-.552246 0-1-.4477-1-1V2c0-.5.399902-1 1-1h18c.5522 0 1 .44772 1 1v16Zm-2-1V6H2v11h16Z"/>
</svg>
```

**Review by ggdouglas — COMMENTED** [review]
Can we clean up these icons a bit? We can do the following:
- Prefer `viewBox` to `width`/`height`
- Remove all `fill` attributes
- Remove all `clip-rule` attributes
- Modify paths to use `nonzero` for `fill-rule`

File: packages/icons/icons.json (hunk 1 above)
Comment (Browun) [reply packages/icons/icons.json:4547]: As discussed, we'll leave these as-is for this PR.

File: resources/icons/16px/th-add.svg (hunk 2 above)
Comment (Browun) [reply resources/icons/16px/th-add.svg:4]: Updated

File: resources/icons/16px/th-list-add.svg (hunk 3 above)
Comment (Browun) [reply resources/icons/16px/th-list-add.svg:3]: Updated

File: resources/icons/16px/th-virtual-add.svg (hunk 4 above)
Comment (Browun) [reply resources/icons/16px/th-virtual-add.svg:3]: Updated

File: resources/icons/16px/th-virtual.svg (hunk 5 above)
Comment (Browun) [reply resources/icons/16px/th-virtual.svg:4]: Updated

File: resources/icons/20px/th-add.svg (hunk 6 above)
Comment (Browun) [reply resources/icons/20px/th-add.svg:4]: Updated

File: resources/icons/20px/th-list-add.svg (hunk 7 above)
Comment (Browun) [reply resources/icons/20px/th-list-add.svg:4]: Updated

File: resources/icons/20px/th-virtual-add.svg (hunk 8 above)
Comment (Browun) [reply resources/icons/20px/th-virtual-add.svg:5]: Updated

File: resources/icons/20px/th-virtual.svg (hunk 9 above)
Comment (Browun) [reply resources/icons/20px/th-virtual.svg:4]: Updated

**Review by ggdouglas — APPROVED** [review]

**Pull request merged**